packages/datetime/src/components/DatePickerCarousel/carouselCalendar/hooks/useWindowResize.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useCallback } from 'react';
+
+export default function useWindowResize(handler, isListening = true) {
```

Comment:
this a hook we will use in other components, I think we should exposed it if we are not :)

Reply:
also, what we are looking for is resize of the container where the component was rendered, that could happen without the window being resized. I know the Table component implements something to detect changes of size in a container.

Reply:
ok i will check... i think i can make a hook, similar to uswWindowResize, that can be useContainerResize... that monitors a Ref and checks if ref.current.clientWidth or ref.current.clientHeight changed

Reply:
also.. we have to check hooks in react-rainbow components that should be exposed as well in order to build components outside the library... like useLocale, useReduxForm.... etc